change/@fluentui-tokens-7b6f098e-f611-4292-adb9-4c0351d335cb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32997&commitId=e4d442c74e09a66d30393d0cedb57cbbe324374a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Events.chromium.png|2| Changed|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>